src/Http/WebUtilities/perf/Microsoft.AspNetCore.WebUtilities.Performance/FormPipeReaderInternalsBenchmark.cs

```diff
@@ -13,8 +13,8 @@ namespace Microsoft.AspNetCore.WebUtilities.Performance
     public class FormPipeReaderInternalsBenchmark
     {
         private byte[] _singleUtf8 = Encoding.UTF8.GetBytes("foo=bar&baz=boo&haha=hehe&lol=temp");
-        private byte[] _firstUtf8 = Encoding.UTF8.GetBytes("foo=bar&baz=boo");
-        private byte[] _secondUtf8 = Encoding.UTF8.GetBytes("&haha=hehe&lol=temp");
+        private byte[] _firstUtf8 = Encoding.UTF8.GetBytes("foo=bar&baz=bo");
```

Comment:
Maybe we should have a 3rd scenario which hits the fast path after the sequence reader finds a single segment span.

Reply:
That's already covered. "foo=bar" will hit the fast path and then "baz=bo" will hit the slow path.